### DISCUSSION TITLE
New Libre 3 version 3.4.2?

### DISCUSSION THREAD

**Discussion by LuisPuertollano** [maheini/FreeStyle-Libre-3-patch#41] [Q&A] [answered]
Couple of days ago, the Libre App was updated to be compatible with Android 13. Is it planed to patch the new version? Details in the Screenshot below, thank you.
![Screenshot_20230226_142538_Nine](https://user-images.githubusercontent.com/77344100/221424936-a791fbb7-2a81-40e6-90ce-e7f05bcace05.jpg)

**Comment by maheini** [comment, marked as answer]
Hi LuisPuertollano 
There are currently no plans of any new releases regarding the patched Libre 3 app, as Juggluco does fully replace any patched app. You can read the instructions for it here: https://github.com/maheini/FreeStyle-Libre-3-patch/blob/main/Juggluco-solution/juggluco-direct-instructions/en/instructions.md

There will come an even easier method (with Juggluco) but it will take some time (~1month)

Maybe there might come a new version, but I wouldn't count on it - and most likely never happen.

**Reply by LuisPuertollano** [reply]
Hi Maheini,

Thank you for your quick response and also for the "spoiler", I'm really thankful for your support. 
I'll take a deeper look to the link you provided. 

Have a nice evening! 
BR

**Comment by Sheihat** [comment]
Dear friend.  I am writing with a request on behalf of the disabled community, with a request to make a patched libre 3.4.2 application, now almost all Samsung phones have stopped transmitting readings.  very necessary

**Reply by maheini** [reply]
Hi @Sheihat 
Luckily, Libre 3 app isn't neccessary anymore - it's absolutely possible to use Juggluco (current version) as your only app! Just scan/start the sensor within Juggluco and your shoulb be ready to go.

Unfortunately, I do not have the time for updating the docs - if you got any spare time, I'd really appreciate a pull request/ edit of the instruction files.

regards, maheini

**Reply by Sheihat** [reply]
can you send a link to this application and a short instruction on how to activate the sensor?  and what to do if the sensor is activated according to the old instructions through the libra 3 application?  10 days left.

**Reply by maheini** [reply]
Here's the link: [Download latest version](https://github.com/maheini/FreeStyle-Libre-3-patch/raw/main/Juggluco-solution/versions/latest/Juggluco.apk)

Steps:
1. Kill the Libre 3 App (within Settings->App... ), in order to disable the bluetooth connection between Libre 3 App and sensor.
2. Download, install, allow all permissions for Juggluco
3. Navigate to settings, click "Libreview" and enter your credentials, click "get account id" and wait (recommended, in order to re-enable/switch sensor between Juggluco - Libre 3 app)
4. (optional) enable Patched Libre Broadcast for xDrip broadcast.
5. Go back to Home screen and just scan the sensor. You should notive a popup and vibration. After started, you will see a notice, informing you about the 60min warmut period on the right side of the Juggluco graph.

Hop it helps - I'm in a hurry, but will try to respond to future questions.
